Version 2.0.0

sergei-fedorov opened the pull request:
Bump version.

joshfriend commented:
I went to upgrade today and could not find info on potentially breaking changes since this is a major version bump.

Everything works the same for me so far, just wanted to make sure I'm not missing anything, so release notes would be appreciated for future releases ❤️ 

I compared the last 2 tags (https://github.com/microsoft/fastlane-plugin-appcenter/compare/v1.11.1...v2.0.0) and it looks like the big difference is the addition of codepush for react native and Ruby 3 support.

sergei-fedorov commented:
Hi @joshfriend! We decided to bump major version since we moved from Ruby 2 to Ruby 3. We added release notes here https://github.com/microsoft/fastlane-plugin-appcenter/releases/tag/v2.0.0
